lizmap/modules/lizmap/lib/Form/QgisFormControl.php

```diff
@@ -708,7 +708,8 @@ public function getStoragePath($layer)
         $targetFullPath = $repPath.$targetPath;
         // Else use given root, but only if it is a child or brother of the repository path
         if (!empty($this->DefaultRoot)) {
-            $fullPath = \Jelix\FileUtilities\Path::normalizePath($repPath.$this->DefaultRoot);
+            // We add $options = 1 to add the trailing slash
+            $fullPath = \Jelix\FileUtilities\Path::normalizePath($repPath.$this->DefaultRoot, 1);
```

Comment:
please use the constant `Path::NORM_ADD_TRAILING_SLASH` , 1 means nothing when reading the code. (and then you can remove the comment)